### PR TITLE
Feature/billing

### PR DESCRIPTION
**Pull request by ygrishajev**

### CODE REVIEW

File: apps/deploy-web/src/utils/walletUtils.ts [hunk 1]
```diff
@@ -7,12 +7,7 @@ export type LocalWalletDataType = {
   name: string;
   selected: boolean;
   isManaged?: boolean;
-};
-
-export const useStorageWallets = () => {
-  const wallets = getStorageWallets();
-
-  return { wallets };
+  userId?: string;
```
Comment (baktun14) [inline apps/deploy-web/src/utils/walletUtils.ts:10]: Maybe I missed it, where do we set this?

File: apps/deploy-web/src/hooks/useManagedWallet.ts [hunk 2]
```diff
@@ -36,6 +36,12 @@ export const useManagedWallet = () => {
     setSelectedNetworkId(NEXT_PUBLIC_MANAGED_WALLET_NETWORK_ID);
   });
 
+  useEffect(() => {
+    if (user?.id && !user.userId) {
```
Comment (baktun14) [inline apps/deploy-web/src/hooks/useManagedWallet.ts:40]: This is for anonymous user account?

**Review by baktun14 — APPROVED** [review]

File: apps/deploy-web/src/utils/walletUtils.ts (hunk 1 above)
Comment (ygrishajev) [reply apps/deploy-web/src/utils/walletUtils.ts:10]: when managed wallet is fetched from the backend it has a userId and is persisted as is to the local storage

File: apps/deploy-web/src/hooks/useManagedWallet.ts (hunk 2 above)
Comment (ygrishajev) [reply apps/deploy-web/src/hooks/useManagedWallet.ts:40]: yes

**Pull request merged**